ScanCommand/src/main/java/io/ballerina/scan/internal/ScanToolConstants.java

```diff
@@ -27,8 +27,11 @@ public class ScanToolConstants {
     static final String SCAN_COMMAND = "scan";
     public static final String RESULTS_JSON_FILE = "scan_results.json";
     static final String BALLERINA_RULE_PREFIX = "B";
-    public static final String PATH_SEPARATOR = "/";
-    public static final String BALLERINA_ORG = "ballerina";
+    static final String PATH_SEPARATOR = "/";
```

Comment:
Shall we use a inbuilt function to get this

Reply:
This constant was meant to be used as a separator for creating the fully qualified rule Identifiers for rules introduced via compiler plugins (i.e.: ballerina/io:B107). Will reword it to something else.